### PR TITLE
add badges to readme

### PR DESCRIPTION
**Pull request by d-m-u**
we're missing these

thanks @Fryguy 

@miq-bot add_label developer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 manageiq-ssh-util
 
+[![Gem Version](https://badge.fury.io/rb/manageiq-ssh-util.svg)](http://badge.fury.io/rb/manageiq-ssh-util)
+[![Build Status](https://travis-ci.com/ManageIQ/manageiq-ssh-util.svg)](https://travis-ci.com/ManageIQ/manageiq-ssh-util)
+[![Code Climate](https://codeclimate.com/github/ManageIQ/manageiq-ssh-util.svg)](https://codeclimate.com/github/ManageIQ/manageiq-ssh-util)
+[![Test Coverage](https://codeclimate.com/github/ManageIQ/manageiq-ssh-util/badges/coverage.svg)](https://codeclimate.com/github/ManageIQ/manageiq-ssh-util/coverage)
+[![Security](https://hakiri.io/github/ManageIQ/manageiq-ssh-util/master.svg)](https://hakiri.io/github/ManageIQ/manageiq-ssh-util/master)
+
+[![Chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ManageIQ/manageiq-ssh-util?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (Fryguy) [inline README.md:9]: We don't have a channel for this gem... You can delete this one

File: README.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+[![Translate](https://img.shields.io/badge/translate-zanata-blue.svg)](https://translate.zanata.org/zanata/project/view/manageiq-ssh-util)
```
Comment (Fryguy) [inline README.md:10]: Also no translation.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
 manageiq-ssh-util
 
+[![Gem Version](https://badge.fury.io/rb/manageiq-ssh-util.svg)](http://badge.fury.io/rb/manageiq-ssh-util)
+[![Build Status](https://travis-ci.com/ManageIQ/manageiq-ssh-util.svg)](https://travis-ci.com/ManageIQ/manageiq-ssh-util)
+[![Code Climate](https://codeclimate.com/github/ManageIQ/manageiq-ssh-util.svg)](https://codeclimate.com/github/ManageIQ/manageiq-ssh-util)
+[![Test Coverage](https://codeclimate.com/github/ManageIQ/manageiq-ssh-util/badges/coverage.svg)](https://codeclimate.com/github/ManageIQ/manageiq-ssh-util/coverage)
+[![Security](https://hakiri.io/github/ManageIQ/manageiq-ssh-util/master.svg)](https://hakiri.io/github/ManageIQ/manageiq-ssh-util/master)
```
Comment (Fryguy) [inline README.md:7]: No hakiri either

**Comment by djberg96** [comment]
Badges?!

**Pull request merged**